client/webserver/site/src/js/forms.ts

```diff
@@ -1598,10 +1598,11 @@ export class LoginForm {
       Doc.show(page.errMsg)
       return
     }
-    if (res.notes) {
-      res.notes.reverse()
+    const notesRes = await getJSON('api/getnotes')
```

Comment:
Looks like a leading `/` is missing, but consider if this separate endpoint is even needed as per my other comment.